src/lib/support/crypto/EllipticCurve.h

```diff
@@ -77,7 +84,7 @@ class EncodedECDSASignature
 public:
     enum
     {
-        kMaxValueLength = ((WEAVE_CONFIG_MAX_EC_BITS + 7) / 8) + 1
+        kMaxValueLength = (Platform::RoundUp(WEAVE_CONFIG_MAX_EC_BITS, 8) / 8) + 1
```

Comment:
It looks like some of the vestigal, non-macro math got left behind here (that is, "/ 8) + 1").

Reply:
RoundUp - rounds up to the 8th multiple.
"/ 8" - converts from bits to bytes
"+ 1" - one extra byte might be needed when value is encoded

Reply:
I'd recommend using NBBY from types.h for the "/ NBBY" part and then, since this math comes up twice, create a macro to the effect of `MaxValueLengthFromBits(aBits)` or some such where you can then also document the +1.